data/models/mongooseResolvers.js

```diff
@@ -0,0 +1,61 @@
+// This file has resolvers for queries that get data from MongoDB using Mongoose.
```

Comment:
I would make this file a `Publication` class with static methods `getOne`, `list`, `getByDOI`, etc.